### PR TITLE
climbing things does not stop tail wagging

### PR DESCRIPTION
**Pull request by Sniblet**
here we go again

# Document the changes in your pull request

Tested, works:
Climbing structures (tables) and machines (???) does not stop tail wagging.

Someday soon:
[Slipping does not stop tail wagging](https://github.com/yogstation13/Yogstation/pull/13452). Left out of this PR because `I have no idea how slipping works` I mean I'm atomizing.

# Changelog

:cl: 
bugfix: climbing things doesn't discourage waggers
tweak: felinids can now oppress other species
/:cl:

### CODE REVIEW

**Review by tattax — CHANGES_REQUESTED** [review]
define H by using var/mob/living/carbon/human/H

**Comment by Sniblet** [comment]
I don't know what that means

**Comment by Sniblet** [comment]
Okay I have no idea what to do about the slip thing
I think I'll just go back on that and pretend I'm "atomizing" or something similarly responsible until I encounter someone else's idea for a fix

File: code/game/machinery/_machinery.dm [hunk 1]
```diff
@@ -629,7 +629,13 @@ Class Procs:
 									"<span class='notice'>You climb onto [src].</span>")
 				log_combat(user, src, "climbed onto")
 				if(climb_stun)
+					var/mob/living/carbon/human/H = user
+					var/wagging = FALSE
+					if(H.dna.species.is_wagging_tail())
```
Comment (ToasterBiome) [inline code/game/machinery/_machinery.dm]:
```suggestion
					if(H && H.dna.species.is_wagging_tail())
```

File: code/game/objects/structures.dm [hunk 2]
```diff
@@ -84,7 +84,13 @@
 									span_notice("You climb onto [src]."))
 				log_combat(user, src, "climbed onto")
 				if(climb_stun)
+					var/mob/living/carbon/human/H = user
+					var/wagging = FALSE
+					if(H.dna.species.is_wagging_tail())
```
Comment (ToasterBiome) [inline code/game/objects/structures.dm]:
```suggestion
					if(H && H.dna.species.is_wagging_tail())
```

**Review by ToasterBiome — DISMISSED** [review]
this is a weird way to do it, also you better check if they are indeed a living/carbon/human for safety™

**Comment by KittyNoodle** [comment]
Not a fix :P

**Comment by Sniblet** [comment]
If you know a better way you’re welcome to overwrite everything, none of this is my own work I just make the PRs

**Review by adamsong — APPROVED** [review]

**Review by JamieD1 — APPROVED** [review]

**Comment by Cartlord** [comment]
Why are you merging furry shit?

**Pull request merged**